src/simstate.jl

```diff
@@ -122,6 +122,9 @@ function initialfeed!(state::SimState, num_initial_infections::Integer, strain::
     push!(state.queue, event)
   end
 
+  event = Event(Val(OutsideTransmissionEvent), 0.0)
```

Comment:
I think it is better to replace this function with the extended version and allow the used to choose which one should be used

Reply:
if the travel function is not used the event will only be triggered once and will do nothing. The problem will disappear after converting the event `OutsideTransmissionEvent` to a function.